.github/workflows/pull_push_request.yml

```diff
@@ -0,0 +1,43 @@
+# This is a CI workflow file created to execute on push and pull requests
+
+name: pull_push_workflow
+on: [pull_request, push]
+jobs:
+
+  kotlin-compiler:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update packages
+        run: sudo apt update
+      - name: Installing JDK
+        run: sudo apt install default-jdk
+      - name: Downloading kotlin compiler
+        run: sudo snap install --classic kotlin
```

Comment:
Install latest version of kotlin using SDKMAN

1. First install SDKMAN using ` curl -s "https://get.sdkman.io" | bash `
2. Install kotlin using sdkman `sdk install kotlin`

Find more info here - https://kotlinlang.org/docs/command-line.html#sdkman

Reply:
I had previously tried this approach but it throws "sdk: command not found error"
![image](https://user-images.githubusercontent.com/93983185/229353439-a8a77bb1-3ef1-4205-8dc9a83cacf49c98.png)

Rest everything works fine.

Reply:
Image: 
![image](https://user-images.githubusercontent.com/93983185/229353510-ab078bc7-8e0c-4c8d-9c9b-35600a1ac373.png)